plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -242,3 +304,53 @@ def process_main(self, item=None):
 
         if new_tag:
             repository.associate_single_unit(self.parent.repo.repo_obj, new_tag)
+
+
+class AddBlobs(PluginStep):
+    """
```

Comment:
what about to make a more general name since we are working here not just with blobs but also manifests?
for example AddUnits?